File: Tracker.API/Features/Identity/Models/LoginResponseModel.cs
using Newtonsoft.Json;

namespace Tracker.API.Features.Identity.Models
{
    public class LoginResponseModel
    {
        public string UserId { get; set; }

        public string Username { get; set; }

        public string Email { get; set; }

        public string Token { get; set; }

        [JsonIgnore] public string RefreshToken { get; set; }
    }
}

File: Tracker.Client/src/auth/models/LoginResponseModel.ts
export class LoginResponseModel {
    constructor(
        readonly _userId: string,
        readonly _username: string,
        readonly _email: string,
        readonly _token: string
    ) {}

    get userId(): string {
        return this._userId;
    }

    get username(): string {
        return this._username;
    }

    get email(): string {
        return this._email;
    }

    get token(): string {
        return this._token;
    }
}